planning/behavior_path_planner/src/utils/lane_change/utils.cpp

```diff
@@ -334,7 +334,7 @@ bool hasEnoughLength(
     const auto lane_changing_time =
       PathShifter::calcShiftTimeFromJerk(shift_length, lateral_jerk, max_lat_acc);
     minimum_lane_change_length_to_preferred_lane +=
-      minimum_lane_changing_velocity * lane_changing_time + common_parameter.minimum_prepare_length;
+      minimum_lane_changing_velocity * lane_changing_time;
```

Comment:
Similar to the utils.cpp, you still need to add minimum prepare length as
`minimum_prepare_length =  0.5 * common_param.max_acc * common_param.lane_change_prepare_duration * common_param.lane_change_prepare_duration`

Reply:
added in [c4c1405](https://github.com/autowarefoundation/autoware.universe/pull/4242/commits/c4c140511a6a695224eef58b828a45c232dcfab4)